rust/tools/authorization-logic/src/ast.rs

```diff
@@ -54,6 +54,13 @@ impl PartialEq for AstPredicate {
 // See https://doc.rust-lang.org/std/cmp/trait.Eq.html
 impl Eq for AstPredicate {}
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct AstBinaryOperation { 
```

Comment:
nit: Name could reflect it is an arithmetic operator. e.g., `AstArithmeticComparison` (Similarly for others related structs.)

Reply:
fixed